IBAnimatable/Animatable.swift

```diff
@@ -22,12 +22,12 @@ public protocol Animatable: class {
   /**
    Animation duration (in seconds)
    */
-  var duration: Double { get set }
+  var duration: TimeInterval { get set }
```

Comment:
Although `public typealias TimeInterval = Double`, It is great to change it to `TimeInterval` 👍. And shall we change all conforming classes to reflect this change to make the code more understandable? If we do, we need to apply to `var delay: TimeInterval { get set }` as well.